### DISCUSSION TITLE
blog/revamp-your-old-android-phone-into-a-mini-linux-server/

### DISCUSSION THREAD

**Discussion by giscus** [AkashRajpurohit/AkashRajpurohit#71] [Comments]
# blog/revamp-your-old-android-phone-into-a-mini-linux-server/

Turn your old Android phone into a portable Linux server with the Linux Deploy app. Learn how to set up a Debian server on your device and use it to host websites, run scripts, and more!

https://akashrajpurohit.com/blog/revamp-your-old-android-phone-into-a-mini-linux-server/

**Comment by xlplex** [comment]
Sadly it doesnt work for me after mounting the container it just says fail :(

**Reply by AkashRajpurohit** [reply]
Can you share some more details about the failure scenario?
I had this working for a long time and it worked perfectly fine.
